src/containers/portal/citizenship/index.tsx

```diff
@@ -449,7 +450,7 @@ function GetCitizenship({ defaultAccount }) {
           address,
           CONTRACT_ADDRESS_PASSPORT,
           msgObject,
-          txs.calculateFee(500000, gasPrice),
+          Soft3jsMsgs.fee(500000),
```

Comment:
Please create const in separate file, no magic numbers, especially for repeatable consts